src/ZulipMobile.js

```diff
@@ -21,17 +21,17 @@ if (config.enableSentry) {
 }
 
 export default () => (
-  <StoreProvider>
-    <TranslationProvider>
-      <StylesProvider>
-        <CompatibilityChecker>
-          <AppEventHandlers>
-            <AppDataFetcher>
+  <CompatibilityChecker>
+    <StoreProvider>
+      <AppEventHandlers>
+        <AppDataFetcher>
+          <TranslationProvider>
+            <StylesProvider>
```

Comment:
Theme change is most frequent, so I thinks it will better to keep it as close as our layout components.
Also by this theme change will not effect the app handlers.
(might be a fix for app doesn't close after pressing back button an Android even from home view, 2-3 press are required.)